File: Day-1(basic)problem/Even_Or_Odd/Even_Or_Odd.java
class EvenOrOdd {
    // Function to check if the number is even or odd
    public static boolean isEven(int n) {
        return n % 2 == 0;
    }

    // Driver Code
    public static void main(String[] args) {
        int n = 15;

        if (isEven(n))
            System.out.println("Even");
        else
            System.out.println("Odd");
    }
}

File: Day-1(basic)problem/Even_Or_Odd/Even_Or_Odd.c
#include <stdio.h>
#include <stdbool.h>

bool isEven(int n)
{
    return n % 2 == 0;
}

int main()
{
    int n = 15;

    if (isEven(n))
        printf("Even\n");
    else
        printf("Odd\n");

    return 0;
}